src/elements/content-sidebar/versions/StaticVersionsSidebar.scss

```diff
@@ -0,0 +1,82 @@
+@import '../../../styles/variables';
+@import '../../common/variables';
+
+.bcs-static-Versions {
+    position: relative;
+    width: 341px; // 1px extra to account for the margin left
+    margin-left: -1px; // -1px to make nav button hover not have blank space
+    border-left: 1px solid $bdl-gray-10;
+
+    .bcs-static-content-header {
+        display: flex;
+        align-items: center;
+        height: 60px;
+        margin: 0 25px;
+        border-bottom: 1px solid $bdl-gray-10;
+
+        .bcs-static-title {
+            display: flex;
+            font-size: 16px;
+        }
+    }
+
+    .bcs-static-scroll-content-wrapper {
+        position: absolute;
+        width: calc(100% - 20px);
+
+        .bcs-static-Versions-content {
+            padding-left: 25px;
+        }
+    }
+}
+
+.bcs-static-Versions-content {
+    pointer-events: none;
+
+    button[data-resin-iscurrent='true'].bcs-VersionsItemButton {
+        width: 100%;
+        background-color: $hover-blue-background;
+        border: 1px solid $bdl-box-blue;
+    }
+}
+
+.bcs-upsell-wrapper {
+    position: relative;
+    height: 100%;
+    text-align: center;
+    background: linear-gradient(to bottom, rgba(255, 255, 255, 0) 0%, rgba(255, 255, 255, 1) 30%);
+
+    .bcs-version-upsell {
+        position: absolute;
+        top: 30%;
+        padding: 48px;
+
+        .bcs-version-upsell-icon {
+            width: 80px;
+            height: 80px;
+        }
+
+        .bcs-upgrade-now {
+            margin: 0 0 10px;
+            font-weight: bold;
+        }
+
+        .bcs-upgrade-button {
+            margin-top: -8px;
+            padding: 10px 15px;
```

Comment:
Do we have any variables we can use for the numeric values in this file?

Reply:
not really, they were values from the css for the non static version.. I could put them in variables, but not really sure how helpful that would be.